comp/process/bundle.go

```diff
@@ -47,10 +48,10 @@ func Bundle() fxutil.BundleOptions {
 		rtcontainercheckimpl.Module(),
 		processdiscoverycheckimpl.Module(),
 
+		agentimpl.Module(),
+
 		hostinfoimpl.Module(),
 		expvarsimpl.Module(),
-		statusimpl.Module(),
-		coreStatusImpl.Module(),
```

Comment:
These are only used by the api server from the original PR: https://github.com/DataDog/datadog-agent/pull/23518
This was moved into main_common where it is supplied.